cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -113,8 +113,8 @@ set(config_module_list
 	#
 	# Logging
 	#
-	#modules/logger
-	modules/sdlog2
+	modules/logger
```

Comment:
We should not change unrelated system defaults as part of a PR - can you revert this?